community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/codegen/ir/ProjectionInstruction.scala

```diff
@@ -24,100 +24,116 @@ import org.neo4j.cypher.internal.compiler.v2_3.codegen.JavaUtils.JavaTypes.{DOUB
-  def literal(value: String): ProjectionInstruction = ProjectLiteral(JavaSymbol( s""""$value"""", STRING))
-  def literal(value: Boolean): ProjectionInstruction = ???
+
+  def literal(value: String): ProjectionInstruction = ProjectLiteral(JavaSymbol(s""""$value"""", STRING))
+
   def parameter(key: String): ProjectionInstruction = ProjectParameter(key)
 
   def add(lhs: ProjectionInstruction, rhs: ProjectionInstruction): ProjectionInstruction = ProjectAddition(lhs, rhs)
+
   def sub(lhs: ProjectionInstruction, rhs: ProjectionInstruction): ProjectionInstruction = ProjectSubtraction(lhs, rhs)
 }
 
-case class ProjectNodeProperty(id: String, token: Option[Int], propName: String, nodeIdVar: String, namer: Namer) extends ProjectionInstruction {
+case class ProjectNodeProperty(id: String, token: Option[Int], propName: String, nodeIdVar: String, namer: Namer)
+  extends ProjectionInstruction {
+
   private val propKeyVar = token.map(_.toString).getOrElse(namer.newVarName())
   private val methodName = namer.newMethodName()
 
   def generateInit() = if (token.isEmpty)
-      s"""if ( $propKeyVar == -1 )
-         |{
-         |$propKeyVar = ro.propertyKeyGetForName( "$propName" );
-         |}
+    s"""if ( $propKeyVar == -1 )
+                          |{
+                          |$propKeyVar = ro.propertyKeyGetForName( "$propName" );
+                                                                               |}
        """.stripMargin
-      else ""
-
-  override def _importedClasses() =
-    Set("org.neo4j.kernel.api.properties.Property", "org.neo4j.kernel.api.exceptions.EntityNotFoundException")
+  else ""
 
   override def members() = if (token.isEmpty) s"private int $propKeyVar = -1;" else ""
 
   def projectedVariable = JavaSymbol(s"$methodName( $nodeIdVar )", OBJECT)
 
-  override protected def _exceptions() = Set(KernelExceptionCodeGen)
+  override protected def importedClasses =
+    Set("org.neo4j.kernel.api.properties.Property", "org.neo4j.kernel.api.exceptions.EntityNotFoundException")
+
+  override protected def exceptions = Set(KernelExceptionCodeGen)
 
-  override def operatorId = Some(id)
+  override protected def operatorId = Some(id)
 
-  override protected def _method: Option[Method] = Some(new Method() {
+  override protected def method = Some(new Method() {
     override def name: String = methodName
 
     override def generateCode: String = {
       val eventVar = "event_" + id
       s"""
-        |private Object $methodName( long nodeId ) throws EntityNotFoundException
-        |{
-        |try ( QueryExecutionEvent $eventVar = tracer.executeOperator( $id ) )
-        |{
-        |$eventVar.dbHit();
-        |$eventVar.row();
-        |return ro.nodeGetProperty( nodeId, $propKeyVar ).value( null );
-        |}
-        |}
+         |private Object $methodName( long nodeId ) throws EntityNotFoundException
+                                      |{
```

Comment:
I'm guessing this is wrong. IntelliJ has a habit of butchering multi-line strings